Cleaned icons

okineadev opened the pull request:
I removed unnecessary properties like "color", "fill-rule", "class", etc.
Also minimizes some icons

okineadev commented:
Do not panic I will save you1!

okineadev commented:
I have to fix the icons changelog, diff, document, lib, log, settings and maybe wakatime_light

But I'm sorry, I don't have electricity right now and my laptop is dead 😢

So when the electricity comes back, I will fix the icons

okineadev commented:
I have to fix the icons changelog, diff, document, lib, log, settings and maybe wakatime_light

But I'm sorry, I don't have electricity right now and my laptop is dead 😢

So when the electricity comes back, I will fix the icons

PKief commented:
Thanks for letting us know. Windicss and processing look a bit strange too. Please make sure that we don't miss anything

okineadev commented:
Okay

okineadev commented:
This can be fixed with `fill="none"`

okineadev commented:
An interesting fact: there were many icons in the repository that do not correspond Material Design 🌚

But I already fixed them

okineadev commented:
An interesting fact: there were many icons in the repository that do not correspond Material Design 🌚

But I already fixed them

okineadev commented:
I have to fix [aurelia](icons/aurelia.svg) icon

![image](https://github.com/material-extensions/vscode-material-icon-theme/assets/81070564/1f8748b7-abe3-4e06-835d-9f24301b9afc)

okineadev commented:
Now I can state that I have carefully reviewed all the icons and fixed the wrong ones, now let's look at the icons (which I adjusted to **Material Design**) whether they are suitable or not

okineadev commented:
@PKief, What do you think of the [Jenkins](icons/jenkins.svg) icon?

[![Jenkins icon](https://raw.githubusercontent.com/material-extensions/vscode-material-icon-theme/a72a18039b1543eaa4f505fc0b2bcebf1db608f6/icons/jenkins.svg)](icons/jenkins.svg)

okineadev commented:
Look at the [Vue](icons/vue.svg) icon. In order to match the **Material Desing**, I had to change the colors of the icon, but as you can see, they don't look good, and in my opinion it is **much better** to leave the **official** icon with the **official** colors. Even if the color checker will argue, you can make the PR merge

[![Vue icon diff](https://github.com/material-extensions/vscode-material-icon-theme/assets/81070564/8dccdbba-eff4-4490-9383-163f7a3c3201)](icons/vue.svg)

PKief commented:
> @PKief, What do you think of the [Jenkins](icons/jenkins.svg) icon?
> 
> [![Jenkins icon](https://raw.githubusercontent.com/material-extensions/vscode-material-icon-theme/a72a18039b1543eaa4f505fc0b2bcebf1db608f6/icons/jenkins.svg)](icons/jenkins.svg)

Better leave it as it is. It's too hard to change all the colors there and ensure that it's still good looking 😅

PKief commented:
> Look at the [Vue](icons/vue.svg) icon. In order to match the **Material Desing**, I had to change the colors of the icon, but as you can see, they don't look good, and in my opinion it is **much better** to leave the **official** icon with the **official** colors. Even if the color checker will argue, you can make the PR merge
> 
> [![Vue icon diff](https://github.com/material-extensions/vscode-material-icon-theme/assets/81070564/8dccdbba-eff4-4490-9383-163f7a3c3201)](icons/vue.svg) 

I had the same feeling when I added this icon to the theme in the beginning. Keeping the original color is fine 👍

okineadev commented:
Great, but how to cancel changes to those icons?

PKief commented:
> Great, but how to cancel changes to those icons?

Can you revert it? Just use the old icons from the main branch and paste them into your branch. It should do the trick

okineadev commented:
> > Great, but how to cancel changes to those icons?
> 
> Can you revert it? Just use the old icons from the main branch and paste them into your branch. It should do the trick

Okay 👌

okineadev commented:
Oh... Bun african edition 😅🧑🏿‍💻

![image](https://github.com/material-extensions/vscode-material-icon-theme/assets/81070564/196cb8d8-9b82-41fd-951e-dfd4336441e0)

okineadev commented:
Need rollback Pug icon

okineadev commented:
Perfect ✨

<img src="https://github.com/material-extensions/vscode-material-icon-theme/assets/81070564/702aed6c-45c4-49a7-b1e6-7aecf1840e24" alt="GIF" width="300"/>

okineadev commented:
@PKief, What do you say about the [watchman](icons/watchman.svg) icon?

<a href="icons/watchman.svg"><img src="https://raw.githubusercontent.com/okineadev/vscode-material-icon-theme/icons-cleaning/icons/watchman.svg" alt="Watchman icon" width="300"/></a>

okineadev commented:
In short, now you can review each icon and add a comment if something is wrong

okineadev commented on icons/cucumber.svg:
😨

okineadev replied:
Fixed with `fill-rule="evenodd"`

okineadev commented:
Error in workflow

![image](https://github.com/material-extensions/vscode-material-icon-theme/assets/81070564/46a45589-f57f-4878-a105-6ccbedbf3601)

PKief commented:
> Error in workflow
> 
> ![image](https://private-user-images.githubusercontent.com/81070564/346176641-46a45589-f57f-4878-a105-6ccbedbf3601.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjAyMDEyOTAsIm5iZiI6MTcyMDIwMDk5MCwicGF0aCI6Ii84MTA3MDU2NC8zNDYxNzY2NDEtNDZhNDU1ODktZjU3Zi00ODc4LWExMDUtNmNjYmVkYmYzNjAxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzA1VDE3MzYzMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVmOGY5ZmJhZDM5M2QzNGM2Y2NhMGY1MzlkYTA4N2U3MjQzOTI3MzEyOGQ3ZTY0MTEzNjE5ZGIyYjVkMTc2NzUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.aU31wxwLETx_QfwPJT2uuJcJ7gbA4fzXlWMxCqXRXVg)
> 
> @PKief

I'm analyzing this issue, it has something to do with an update of the library which I'm using here.

PKief commented:
> @PKief, What do you say about the [watchman](icons/watchman.svg) icon?
> 
> [<img alt="Watchman icon" width="300" src="https://raw.githubusercontent.com/okineadev/vscode-material-icon-theme/icons-cleaning/icons/watchman.svg">](icons/watchman.svg)

It looks good to me 👍

PKief reviewed (approved):
Thanks a lot, it looks good to me 👍

okineadev commented:
> Perfect ✨
> 
> ![GIF](https://private-user-images.githubusercontent.com/81070564/345903524-702aed6c-45c4-49a7-b1e6-7aecf1840e24.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ4MzU4NjEsIm5iZiI6MTcyNDgzNTU2MSwicGF0aCI6Ii84MTA3MDU2NC8zNDU5MDM1MjQtNzAyYWVkNmMtNDVjNC00OWE3LWIxZTYtN2FlY2YxODQwZTI0LmdpZj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODI4VDA4NTkyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNlMzhlNzdkOTgzY2ViNDcyNDQ4MjIyYTMxZTgzNWM0NTE4NmZiMGMxMDE1MGU1OTIyZDBhMTM2MWE2NzczMjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.MMI6M-tEuD_eyjbUcaXcs_vsw-ET-9jtXk4aru06K6Y)

It's not a spam

okineadev commented:
> Oh... Bun african edition 😅🧑🏿‍💻
> 
> ![image](https://private-user-images.githubusercontent.com/81070564/345754191-196cb8d8-9b82-41fd-951e-dfd4336441e0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ4MzU4NjEsIm5iZiI6MTcyNDgzNTU2MSwicGF0aCI6Ii84MTA3MDU2NC8zNDU3NTQxOTEtMTk2Y2I4ZDgtOWI4Mi00MWZkLTk1MWUtZGZkNDMzNjQ0MWUwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODI4VDA4NTkyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY5ZTYxYTNiYTc3NTY1MDA3YzgxNzhmMTA0Nzg3N2I2MmZmYWQ3MmQwZjY0NTQ5MzdjYmI4OTRjMzM1NDY1MGMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.s9nSD7fjOIq79gSB9e0M3O4-hRuEoQXtoJ0r0WoMp5g)

This is not abuse, I deeply respect and support black people

PKief commented:
> > Oh... Bun african edition 😅🧑🏿‍💻
> > ![image](https://private-user-images.githubusercontent.com/81070564/345754191-196cb8d8-9b82-41fd-951e-dfd4336441e0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ4MzU4NjEsIm5iZiI6MTcyNDgzNTU2MSwicGF0aCI6Ii84MTA3MDU2NC8zNDU3NTQxOTEtMTk2Y2I4ZDgtOWI4Mi00MWZkLTk1MWUtZGZkNDMzNjQ0MWUwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODI4VDA4NTkyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY5ZTYxYTNiYTc3NTY1MDA3YzgxNzhmMTA0Nzg3N2I2MmZmYWQ3MmQwZjY0NTQ5MzdjYmI4OTRjMzM1NDY1MGMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.s9nSD7fjOIq79gSB9e0M3O4-hRuEoQXtoJ0r0WoMp5g)
> 
> This is not abuse, I deeply respect and support black people

The picture showed a black bun logo and you post a laughing emoji with it and associate it with Africa. You can interpret anything from it, good and bad. Therefore, this should be avoided in our posts in this repository. Specifically: Jokes about any kind of physical characteristics, skin color, political views, religion, I don't want to see any of that here.

PKief commented:
> Perfect ✨

Messages like "+1", "ping", "@somebodiesname", "?", etc. which don't add some information or value to a chat can be considered as spam. Expressing emotions can easily be done via emojis on single posts or on the issue's initial comment and is not flooding the inboxes of all watchers. Maybe I'm too strict with the message "Perfect ✨" and it's nice that you liked something. But it could have been expressed via emoji as well.
